mule-module-apikit/src/main/java/org/mule/module/apikit/uri/TokenVariable.java

```diff
@@ -95,7 +95,7 @@ public Pattern pattern()
      */
     public boolean resolve(String expanded, Map<Variable, Object> values)
     {
-        values.put(this._var, URICoder.decode(expanded));
+        values.put(this._var, URICoder.decode(expanded).replaceAll("%2F", "/"));
```

Comment:
percent encoding is case insensitive, so %2f should be converted too, adding "(?i)" at the beginning of the regex may do the trick, add it to the test too pls